R/pkg/R/sparkR.R

```diff
@@ -365,6 +365,10 @@ sparkR.session <- function(
     }
     overrideEnvs(sparkConfigMap, paramMap)
   }
+  if (nzchar(master)) {
+    assign("spark.master", master, envir = sparkConfigMap)
```

Comment:
For example, these two is equivalent before this change:
```
sparkR.session(spark.master = "yarn-client")
```

```
sparkR.session(master = "local", spark.master = "yarn-client")
```

The reason for the explicit `master` parameter is historical - we want to be backward compatible to users of `sparkR.init`. So I think `master` should not overwrite `spark.master`.